package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@aws-cdk/aws-amplify": "^1.98.0",
     "@aws-cdk/aws-appsync": "^1.98.0",
+    "@aws-cdk/aws-codecommit": "^1.99.0",
```

Comment:
Could you use version 1.98.0 instead? We will upgrade to 1.99.0 soon.

Reply:
Ah right, that was an oversight. Now the remaining dependency errors are gone.